spark/src/test/scala/org/apache/spark/sql/delta/test/DeltaTestImplicits.scala

```diff
@@ -91,6 +93,40 @@ object DeltaTestImplicits {
     }
   }
 
+  /** Helper class for working with [[TableCommitCoordinatorClient]] */
```

Comment:
These helpers avoid having to retrofit zillions of call sites in path-based unit tests?

Reply:
Yes - these are very useful.